.github/workflows/infrastructure_tests.yml

```diff
@@ -0,0 +1,59 @@
+name: Test Examples on Remote Infrastructure
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [ opened, synchronize ]
```

Comment:
Could we make this test run only once by removing these triggers here?

Reply:
yes, once everything else works well together